setup.cfg

```diff
@@ -12,6 +12,7 @@ test = pytest
 
 [flake8]
 exclude = .*,env,lib,dist,build
+ignore = E402
```

Comment:
I think it's a hangover from a previous commit. I noticed the flake 8 checks were broken anyway, so am removing the above and fixing that.